modules/virt-maintain-high-availability.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * virt/install/preparing-cluster-for-virt.adoc
+
+[id="virt-maintain-high-availability_{context}"]
+= Maintaining high availability of virtual machines
+
+There are three options to maintain high availability (HA) of virtual machines:
```

Comment:
The main problem with creating a separate module for this topic is the xref issue. Because we cannot use xrefs in modules, I suggest keeping this info in the assembly under a `==` section heading. For example:
````
[id="virt-maintain-high-availability_preparing-cluster-for-virt"]
== How to maintain high availability of virtual machines
````
That would still create the same visual separation.